plantisc2/tasks/php-config.yml

```diff
@@ -0,0 +1,14 @@
+---
+- name: Ensure php7.0-fpm cgi.fix_pathinfo=0
+  lineinfile:
+    dest: /etc/php/7.0/fpm/php.ini
+    regexp: '^(.*)cgi.fix_pathinfo='
+    line: cgi.fix_pathinfo=0
+  notify:
```

Comment:
yes, as long as it's within the same role, you can call the handlers from any task.